File: MovieDB.Api/Entities/Concert.cs
using System;
using System.Text.Json.Serialization;

namespace MovieDB.Api.Entities
{
    public enum ConcertGenre
    {
        Rock,
        Classic,
        Reggae,
        Pop,
        Latin,
        Electro,
        DrumnBass
    }

    public class Concert
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public DateTime SeenAt { get; set; }
        public string Location { get; set; }
        public ConcertGenre Genre { get; set; }
        public Rating Rating { get; set; }
        [JsonIgnore]
        public Account Account { get; set; }
        public DateTime CreatedAt { get; set; }
        public DateTime? UpdatedAt { get; set; }
        public DateTime? DeletedAt { get; set; }
    }
}

File: MovieDB.Client/src/types/concert.ts
export type TConcert = {
  id: number
  title: string
  seenAt: string
  location: string
  genre: number
  rating: number
}

export const concertGenres = [
  { text: 'Rock', value: 0 },
  { text: 'Classic', value: 1 },
  { text: 'Reggae', value: 2 },
  { text: 'Pop', value: 3 },
  { text: 'Latin', value: 4 },
  { text: 'Electro', value: 5 },
  { text: 'DrumnBass', value: 6 },
]
